WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -35,13 +41,23 @@ class LoginPrologueViewController: LoginViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        
         configureButtonVC()
         navigationController?.setNavigationBarHidden(true, animated: false)
     }
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
+        tracker.set(flow: .prologue)
+        
+        if !Self.prologueFlowTracked {
```

Comment:
```suggestion
        if !self.prologueFlowTracked {
```

Reply:
The reason I'm using `Self` here is because I want to reference the class instead of the instance (because `prologueFlowTracked` is static).

In fact if I changed it:

<img width="1111" alt="Screen Shot 2020-09-03 at 11 00 46" src="https://user-images.githubusercontent.com/1836005/92125107-feafb880-edd4-11ea-9eb0-cab30db7ce22.png">

Reply:
I changed the variable to be an instance variable instead.  I figure it's more appropriate.